edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -18,7 +19,7 @@ namespace Microsoft.Azure.Devices.Edge.Agent.IoTHub
 
     public class EdgeAgentConnection : IEdgeAgentConnection
     {
-        internal static readonly Version ExpectedSchemaVersion = new Version("1.0");
+        internal static readonly Version ExpectedSchemaVersion = new Version("1.1.0");
```

Comment:
Per https://github.com/SchemaStore/schemastore/pull/1116/files#diff-25894c152d2d2b3bedcf8f3151fd5193, it looks like the schemaVersion should be `1.1` not `1.1.0`?

Reply:
What is the expected version of EdgeHub?

Reply:
it's 1.1.0 for both EH and EA in code, but it makes no difference for the validation logic, as we only care about major+minor, patch is ignored.